### PR TITLE
Added global filter and exceptions for network tests

### PR DESCRIPTION
**Pull request by kirthik**
@markcowl

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kirthik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kirthik). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: tools/GlobalFilters.xml [hunk 1]
```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="utf-8"?>
+<GlobalFilters xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <ContentSearchFilters>
+    <ContentFilter>
+      <Name>Initial ContentFilters</Name>
+      <Filters>
```
Comment (markcowl) [inline tools/GlobalFilters.xml]: @kirthik Did we have anythign to do with this Filters section, or is this the default from the xml file?  If it's the default, I think we should only store the section with our specific changes.

File: tools/GlobalFilters.xml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        <string>Name=\{\d\}|\[</string>
+        <string>XTestEncrypted</string>       
+      </Filters>
+    </ContentFilter>
+  </ContentSearchFilters>
+  <ResourceSearchFilters>
+    <ResourceFilter>
+      <Name>Mock/UnitTest Path Filters</Name>
+      <Filters>
+        <string>UnitTest</string>
```
Comment (markcowl) [inline tools/GlobalFilters.xml]: Are thesae first 5 ours, or were they in the file before?

File: tools/GlobalFilters.xml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        <string>publickey|pubkey|hash|digestvalue</string>
+        <string>REPLACE_DATABASE_NAME|REPLACE_SERVER_NAME|SQLAZURE\.SERVERNAME|Data\sSource=(\.|\(localDB\))|Server=\.\\|\.DataClasses\.|\(local\)|\|DataDirectory\||dummy|localhost|127\.0\.0\.1|localhost</string>
+        <string>DatabasePublish\.</string>
+        <string>\|SqlExpressInstanceName\|</string>
+        <string>\[SERVERNAME|STORAGE_ACCOUNT_KEY|ACCOUNT_KEY|your\ssecret\|UPDATE\sBEFORE\sDEPLOYMENT]</string>
+        <string>Catalog=TESTDUMMY</string>
+        <string>PLACEHOLDER</string>
+        <string>MyReleaseDB</string>
+        <string>Password="UNUSED"</string>
+		<string>user ID</string>
```
Comment (kirthik) [inline tools/GlobalFilters.xml]: This part is ours

File: tools/GlobalFilters.xml (hunk 2 above)
Comment (kirthik) [inline tools/GlobalFilters.xml]: They were already in the file

File: tools/GlobalFilters.xml (hunk 1 above)
Comment (kirthik) [inline tools/GlobalFilters.xml]: I added `user ID` as a content filter. We had `user=User ID` in one of the tests and it was showing up as a match

**Comment by markcowl** [comment]
@kirthik So, my worry is that, if we blindly overwrite everythign in the globafilters file, we miss out on updates to global filters that may come in new versions of CredSCan.  What do you think of merging in just the xml for our additions with whatever xml we find in the globalfilters file?  This would mean just saving our global changes in this file.

**Comment by kirthik** [comment]
talked to @markcowl in person. The feature he is talking about should be in the tool by end of June. Adding filter content in `GlobalFilters.xml` is just as temporary fix. Hence decided to merge this.

**Pull request merged**